### PR TITLE
Update commerce_price to commerceguys/intl class name changes.

### PR DESCRIPTION
**Pull request by mglaman**

### CODE REVIEW

File: modules/price/src/CurrencyImporter.php [hunk 1]
```diff
@@ -49,7 +49,7 @@ class CurrencyImporter implements CurrencyImporterInterface {
   public function __construct(EntityManagerInterface $entity_manager, LanguageManagerInterface $language_manager) {
     $this->currencyStorage = $entity_manager->getStorage('commerce_currency');
     $this->languageManager = $language_manager;
-    $this->currencyManager = new DefaultCurrencyManager();
+    $this->$currencyRepository = new CurrencyRepository();
```
Comment (bojanz) [inline modules/price/src/CurrencyImporter.php]: we have an extra dollar sign here.

File: modules/price/src/CurrencyImporter.php [hunk 2]
```diff
@@ -57,7 +57,7 @@ public function __construct(EntityManagerInterface $entity_manager, LanguageMana
    */
   public function getImportableCurrencies($fallback = CurrencyImporterInterface::FALLBACK_LANGUAGE) {
     $language = $this->languageManager->getCurrentLanguage();
-    $importable_currencies = $this->currencyManager->getAll($language->getId(), $fallback);
+    $importable_currencies = $this->$currencyRepository->getAll($language->getId(), $fallback);
```
Comment (bojanz) [inline modules/price/src/CurrencyImporter.php]: and here

File: modules/price/src/CurrencyImporter.php [hunk 3]
```diff
@@ -144,7 +144,7 @@ public function importCurrencyTranslations($currencies = array(), $languages = a
    */
   protected function getCurrency($currency_code, LanguageInterface $language, $fallback = NULL) {
     try {
-      $currency = $this->currencyManager->get($currency_code, $language->getId(), $fallback);
+      $currency = $this->$currencyRepository->get($currency_code, $language->getId(), $fallback);
```
Comment (bojanz) [inline modules/price/src/CurrencyImporter.php]: and here

**Comment by mglaman** [comment]
@bojanz all fixed up.

**Pull request merged**